contracts/USM.sol

```diff
@@ -12,7 +14,7 @@ import "./oracles/IOracle.sol";
  * @author Alex Roan (@alexroan)
  * @notice Concept by Jacob Eliosoff (@jacob-eliosoff).
  */
-contract USM is ERC20 {
+contract USM is ERC20, Delegable {
```

Comment:
Do you not need `USM is IUSM`?

Reply:
Not really, and `interface` is a declaration of the methods contained in a contract, but the contract itself doesn't need to inherit the interface for it to be valid. Actually, the contract inheriting the interface makes the code more unreadable (by demanding `override` in each function).

However, to follow general practice, and maybe to get some compiler warnings if something goes wrong, I did `USM is IUSM`.